### ISSUE TITLE
[SF][A11Y][Button] some icon only button svg with role="img" don't have an aria-label

### ISSUE THREAD

**Issue by sfsf-xweb-sh** [SAP/ui5-webcomponents#5596]
### **Bug Description**
some icon only button svg with role="img" don't have an aria-label

### **Expected Behavior**
icon only button svg with role="img" should have an aria-label.

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Button/
2. see the result with chrome dev tool
<img width="1132" alt="Screen Shot 2022-08-01 at 18 17 52" src="https://user-images.githubusercontent.com/78520316/182127549-5e5a61ed-c817-4280-84df-8fc04c07fb43.png">

### **Context**
- UI5 Web Components version: 1.6.0
- OS/Platform: all
- Browser: all
- Affected component: ui5-button

### **Log Output / Stack Trace / Screenshots**
```
{...}
```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: {SF}
- Business impact: {Medium}

**Comment by MapTo0** [comment]
Dear @SAP/ui5-webcomponents-topic-b 

We discusses with Boyan, that you have done some work on that topic with that change: https://github.com/SAP/ui5-webcomponents/pull/5592

Could you please check if it satisfies the requirement here.

Regards,
Martin

**Comment by unazko** [comment]
Hello @sfsf-xweb-sh,

There is the following component enhancement: https://github.com/SAP/ui5-webcomponents/issues/5733
With it we'll apply the default tooltip (coming from the inner icon) to the <code>ui5-button</code> component directly.
This will result in no AXE reports in the described use case as the inner <code>ui5-icon</code> will have role "presentation".

Currently the internal <code>ui5-icon</code> tooltip is displayed for the <code>ui5-button</code> if no <code>tooltip</code> property is set. Some <code>ui5-icon</code>'s do not provide a default tooltip and so
is the icon-only<code>ui5-button</code>, which is uses such icon. There are also cases when the default icon tooltip doesn't describe the button's function well enough. In such cases there should be a <code>tooltip</code> property set at application side.

"As per UX guidelines all icon-only button instances should have a tooltip. The tooltip attribute should be set at application side for all cases when there is no default tooltip coming from the inner ui5-icon element."

Best Regards,
Boyan Rakilovski
